packages/paste-core/components/spinner/__tests__/index.test.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react';
+import {matchers} from 'jest-emotion';
+import {Theme} from '@twilio-paste/theme';
+import {render, screen} from '@testing-library/react';
+import type {RenderOptions} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+
+import {Spinner} from '../src';
+// @ts-ignore typescript doesn't like js imports
+import axe from '../../../../../.jest/axe-helper';
+
+expect.extend(matchers);
+
+const TestWrapper = (elements?: Record<string, any>): RenderOptions['wrapper'] => ({children}) => (
+  <Theme.Provider theme="default">
+    <CustomizationProvider
+      // @ts-expect-error global test variable
+      theme={TestTheme}
+      elements={elements}
+      data-testid="test-wrapper"
+    >
+      {children}
+    </CustomizationProvider>
+  </Theme.Provider>
+);
+
+describe('Spinner', () => {
+  describe('HTML Attribute', () => {
+    it('should set an element data attribute for Spinner (default)', () => {
+      render(<Spinner color="colorText" title="Default name customization" decorative={false} />, {
+        wrapper: TestWrapper(),
+      });
+
+      const nodeNamedSpinner = screen.getByTestId('test-wrapper').firstChild as HTMLElement;
```

Comment:
It could be yeah, but in this case I am saying that I expect the first child to be this thing, so if the tree changes this test will fail. I prefer that since typically unit tests should fail when changes are made, otherwise we could end up with sneaky false positives or missed bugs.